### PR TITLE
Arodriguez/feature 149 hazard report form UI select hazard option

### PR DESCRIPTION
**Pull request by ALFAROSO**
- Select Hazard Option UI 
- Displaying orange icon form previous ticket

### CODE REVIEW

File: pages/report-hazard/style.css [hunk 1]
```diff
@@ -126,8 +125,7 @@ input[type='radio'] {
   background: var(--neutral-50);
 }
 
-#hazard-category input[type='radio']:checked + label,
-#hazard-option-content input[type='radio']:checked + label {
+#hazard-category input[type='radio']:checked + label {
   border-radius: 8px;
   border: 1px solid var(--neutral-100);
   background: var(--secondary-500);
```
Comment (ReinhardtBotha) [inline pages/report-hazard/style.css:131]: Add position: relative here to make the orange icon stick to the option UI card

```suggestion
  background: var(--secondary-500);
  position: relative;
```

Comment (ALFAROSO) [reply pages/report-hazard/style.css:131]: image

Comment (ALFAROSO) [reply pages/report-hazard/style.css:131]: Better now

Comment (ReinhardtBotha) [reply pages/report-hazard/style.css:131]: perfect!

**Review by whamester — APPROVED** [review]

**Pull request merged**